libs/designsystem/src/lib/components/form-field/input/input.component.spec.ts

```diff
@@ -149,3 +149,17 @@ describe('InputComponent', () => {
     });
   });
 });
+
+describe('when configured with size medium', () => {
+  const createHost = createHostFactory({
+    component: InputComponent,
+  });
+  it('should render with correct height', () => {
+    const spectator = createHost(`
+    <input kirby-input size="md"/>
+    `);
+    expect(spectator.element).toHaveComputedStyle({
+      height: size('xl'),
+    });
+  });
```

Comment:
Suggestion: Add test that shows that kirby-input defaults to large size? A test showing there is no breaking changes.